test/test_cli.py

```diff
@@ -107,24 +107,6 @@ def test_invalid_file_or_directory(self):
             main(["cve-bin-tool", "non-existant"])
         assert e.value.args[0] == ERROR_CODES[FileNotFoundError]
 
-    def test_null_byte_in_filename(self):
-        """Test behaviour with an invalid file/directory that contains a \0"""
-
-        # Put a null byte into the filename of a real file used in other tests
-        CSV_PATH = Path(__file__).parent.resolve() / "csv"
-        null_byte_file = str(CSV_PATH / "test_triage\0.csv")
-
-        # for Python 3.8+ this should raise FileNotFound
-        if sys.version_info.major == 3 and sys.version_info.minor > 7:
-            with pytest.raises(SystemExit) as e:
-                main(["cve-bin-tool", null_byte_file])
-            assert e.value.args[0] == ERROR_CODES[FileNotFoundError]
-
-            null_byte_file = str(CSV_PATH / "test_triage.csv\0something")
-            with pytest.raises(SystemExit) as e:
-                main(["cve-bin-tool", null_byte_file])
-            assert e.value.args[0] == ERROR_CODES[FileNotFoundError]
-
```

Comment:
I don't think this function should be removed. You only need to remove condition checking, it was always executing on Python 3.8+.